.github/workflows/job_sonarqube.yml

```
@@ -24,6 +24,8 @@ jobs:
       run:
         shell: bash
         working-directory: ${{ inputs.WORKING_DIRECTORY }}
+    env:
+      CONTINUE_ON_ERROR: ${{inputs.CONTINUE_ON_ERROR}}
```

nitpick: formatting inconsistent with line 26

```suggestion
      CONTINUE_ON_ERROR: ${{ inputs.CONTINUE_ON_ERROR }}
```